### PR TITLE
chore: Update setup instructions to prefer docker

### PR DESCRIPTION
**Pull request by lili2311**
### What change does this PR introduce?

Update the README with most up to date setup advice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,16 +43,49 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
...
-Install virtualenv: pip install virtualenv
-Install virtualenvwrapper: pip install virtualenvwrapper \*You can skip this and use virtualenv installed in a step before directly, virtualenvwrapper allows for nice interfacing with virtualenv
-Source the virtualenvwrapper: source /usr/local/bin/virtualenvwrapper.sh NOTE: To help do this automatically on every new shell you open add the line above to your .bash_profile or .bashrc
-Create a new env for the project: mkvirtualenv pimp
+## Project Setup
 
-## Laptop Setup
+1. Clone the repo: `git@github.com:PolyglotDevsLondon/sharespotsgit`
+1. [Download Docker](https://www.docker.com/products/docker-desktop)
+2. From the root of your project run:
```
Comment (padamczewski) [inline README.md]: Github will numerate the list items out of the box
```suggestion
1. From the root of your project run:
```

File: README.md [hunk 2]
```diff
@@ -43,16 +43,49 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
...
+```
+docker-compose -f local.yml build
+```
+3. Then run:
```
Comment (padamczewski) [inline README.md]: Github will numerate the list items out of the box

```suggestion
1. Then run:
```

File: README.md [hunk 3]
```diff
@@ -43,16 +43,49 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
...
+
+```
+docker-compose -f local.yml up
+```
+4. Make sure you dont have any errors :)
```
Comment (padamczewski) [inline README.md]: Github will numerate the list items out of the box

```suggestion
1. Make sure you don't have any errors :)
```

File: README.md [hunk 4]
```diff
@@ -43,16 +43,49 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
...
+
+## Front End changes
+
+1. To install [Sass](https://sass-lang.com/install)
+
+- You can install using node package manager `docker-compose -f local.yml run npm install -g sass`
+- Windows: `choco install sass`
+- Mac OS X: `brew install sass/sass/sass`
+
+2. To build everything Front end related run `npm run build`
```
Comment (padamczewski) [inline README.md]: Github will numerate the list items out of the box
```suggestion
1. To build everything Front end related run `npm run build`
```

File: README.md [hunk 5]
```diff
@@ -43,50 +43,33 @@ Simple Django app that displays a list of curated cafes/spaces to meetup and wor
...
-7. Apply the initial database migrations: `python manage.py migrate`
+Included is django, postgres, pgadmin and jupyter notebook run from django_extensions
 
-## Running the project locally
+# Running the project locally
 
-0. If not already active, activate the virtual environment: `workon sharespots`
-1. Run `python manage.py runserver`
-1. Open your browser at http://localhost:8000/
+1. Run `docker-compose -f local.yml run django python manage.py runserver`
```
Comment (simonRedwards) [inline README.md]: For me, just running docker "up" (line 57) starts the server too, on 0.0.0.0:8000

**Pull request merged**